src/components/WwwFrame/LendMenu/TheLendMenu.vue

```diff
@@ -55,6 +63,20 @@ export default {
 		LendListMenu,
 		LendMegaMenu,
 	},
+	created() {
+		const { enabled } = getExperimentSettingCached(this.apollo, lendMenuExpKey);
```

Comment:
I think this should all be within an `!this.$isServer` check here so it only runs in the browser. Otherwise it will try to run during server render but likely not complete.

The other usage of `getExperimentSettingAsync` in the Featured Hero is in the `prefetch` function which is specifically built to resolve on the server and sets up the cached data for the created method in that context.

Being that the lend menu is hidden on page load, I don't think we need to add this to prefetch but keeping it in created should ensure it's ready when someone hovers over or activates the menu...

Reply:
Thank Matt, I'll add that flag to ensure is running in the browser.

I saw the other usage of `getExperimentSeetingsAsync`, I tried to put it in the `prefetch` method but it was being ignored

Reply:
@mcstover using preFetch doesn't work in this case because the TheLendMenu component is asynchronously loaded in TheHeader, so it isn't found by the getDeepComponents function of the preFetch process.

Reply:
Right on. As mentioned above, I don't think we should use prefetch for this. I just also think we should prevent it from initiating a promise during the server render that may or may not complete.

Reply:
`TheLendMenu: () => import('@/components/WwwFrame/LendMenu/TheLendMenu'),` would def have an ill affect on prefetch but not really b/c we intentionally loaded it this way.